src/App.tsx

```diff
@@ -12,6 +13,7 @@ class App extends React.Component {
         <div>
```

Comment:
The div element is not necessary.

Reply:
Actually, the div shouldn't be needed since the routes are already encapsulated by the parent Router element. It is not the question about who did it, it just the matter that it is an unnecessary element in this while and should be removed.

Reply:
I still do not agree that it doesn't matter who did it and that it's not a part of this PR. When doing a PR review i think that it is very important to focus only on the changes that are made in the specific PR that you are reviewing. of course you could always find other things that you think should change, like this div for example. but in these cases i think you instead should talk to the team about it outside the PR review and/or create a new bug issue on it so someone could take a look on it. 
But just to prove my point from before i will now show you that this div can't just be removed. 

I guess that we are talking about the same div since there is only one in this file "App.tsx"
![bild1](https://user-images.githubusercontent.com/31758737/50681670-a414f680-100c-11e9-9ba7-de0ffda969fe.png)

by removing this div you will get this.
![ild2](https://user-images.githubusercontent.com/31758737/50681689-b2631280-100c-11e9-9088-e076a8421c0e.png)

You are welcome to try this on your own in another branch since you will most likely have this div there to if there are more than one route path. 

You are also most welcome to provide another solution for this, but since this still does not have anything to do with this PR and branch you should create another issue and branch for that. 

Sincerely / Filip

Reply:
The whole router system will be placed in an other issue since it needs major rework and structure differently. But pr are also a code review to optimize the application. For example the index.tsx file need to wrap the <App> example: 

`ReactDOM.render(
  <BrowserRouter>
    <App />
  </BrowserRouter>,
  document.getElementById('root') as HTMLElement
);`

and in the file App.tsx we need a switch instead. JUst a simple solution anyway this will not be placed here afterwards.

`public render() {
    return (
      <Switch>
          <Route path='/' exact={true} component={Lobby} />
          <Route path='/game/:id' component={Game} />
          <Route path='/await/:id' component={AwaitGame} />
      </Switch>
    );`

Needs more clear specific mention in the pr then that 'do not review the router system'.